MongoDB/node.updateOne.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+    logger.trace("MongoDB UpdateOne Node: Invoked!");
+    try {
+        logger.trace('MongoDB updateOne Node: Input - ${JSON.stringify(inputData)}');
+
+        if (_.isEmpty(inputData.filter)) {
+            throw new Error('Filter is empty');
+        }
+        if (_.isEmpty(inputData.update)) {
+            throw new Error('Update data is empty');
+        }
+
+        const result = await connectorData.db.collection(inputData.collection).updateOne(inputData.filter, { $set: inputData.update });
```

Comment:
MongoDB update operation has an an upsert option. We should have an input option for upsert and that should be send to the update operation